src/omnicore/tx.cpp

```diff
@@ -82,6 +82,16 @@ bool CMPTransaction::isOverrun(const char* p)
 
 // -------------------- PACKET PARSING -----------------------
 
+/** Temporarily disables packet debugging and then calls interpret_Transaction(). */
+bool CMPTransaction::reinterpret_Transaction()
+{
+    bool packetDebugState = msc_debug_parser;
+    msc_debug_parser = 0;
```

Comment:
Ahh, that's a very good point - thanks :)

Shoot - now to find another way to skin this proverbial cat...